src/hooks/index.ts

```diff
@@ -20,5 +20,6 @@ export * from './useTokenApproval';
 export { default as useUniqueId } from './useUniqueId';
 export { default as useWallet } from './useWallet';
 export { default as useRanking } from './useRanking';
-export { default as useCurrentSeason } from './useCurrentSeason/useCurrentSeason';
+export { default as useCurrentSeason } from './season/useCurrentSeason';
+export { default as usePastSeasons } from './season/usePastSeasons';
```

Comment:
yes usePastSeasons is utility that will return all seasons that have passed. Right now it will return only OG season as passed one. In your example Season1 and Season OG once Season 2 is active.

On the second topic, the more we split the more maintainable it is. The files are still under the "utilities/season" but they are serving different purpose. I kinda don't want my future react components to have to import big chunks of code only to use a small bit of that code. I don't think it is cleaner since if I have to think about writing a test for the bigger service you propose to my head it already seems as "big" and "messy".